Docs/Getting Started Guide.md

```diff
@@ -152,28 +140,29 @@ struct SetOAuthURL: Action {
 
 ## Reducers
 
-Reducers are the only place in which you should modify application state! Reducers take the current application state and an action and return the new transformed application state. We recommend to provide many small reducers that each handle a subset of your application state.
+Reducers are the **only place** in which you should **modify** application state. Reducers take the current application state and an action then return the new transformed application state. We recommend to provide many small reducers that each handle a subset of your application state.
 
-You can do this implementing a top-level reducer that conforms to the `Reducer` protocol. This reducer will then call individual functions for each different part of the app state.
+You can do this by implementing a top-level reducer that conforms to the `Reducer` protocol. This reducer will then call individual reducer functions for each different part of the app state, according to their areas of responsibility.
```

Comment:
Excellent changes so far! While we're at it, there's no protocol anymore, just a typealias for the function signature. You may want to change the wording accordingly.